Description: Fix build failures by detecting and linking libraries:
 - OPENGL
 - Threads
 - EXPAT

Author: Gianfranco Costamagna <locutusofborg@debian.org>

Index: fgrun-3.4.0.final/CMakeLists.txt
===================================================================
--- fgrun-3.4.0.final.orig/CMakeLists.txt
+++ fgrun-3.4.0.final/CMakeLists.txt
@@ -133,6 +133,9 @@
 
 find_package(Boost REQUIRED)
 find_package(ZLIB REQUIRED)
+find_package(Threads REQUIRED)
+find_package(EXPAT REQUIRED)
+
 if (SHARED_FLTK)
     #see http://sources.debian.net/src/fltk1.3/1.3.2-4/README.CMake.txt lines 200-240
     find_package(FLTK REQUIRED NO_MODULE)
@@ -143,6 +146,7 @@
 endif (SHARED_FLTK)
 find_package(OpenSceneGraph 2.8.1 REQUIRED osgText osgDB osgParticle osgUtil osgViewer osgGA)
 find_package(SimGear ${FGRUN_VERSION} REQUIRED)
+find_package(OpenGL REQUIRED)
 if (ENABLE_NLS)
     find_package(Gettext)
 endif (ENABLE_NLS)
Index: fgrun-3.4.0.final/src/CMakeLists.txt
===================================================================
--- fgrun-3.4.0.final.orig/src/CMakeLists.txt
+++ fgrun-3.4.0.final/src/CMakeLists.txt
@@ -74,6 +74,8 @@
     ${PLATFORM_LIBS}
     ${OPENGL_LIBRARIES}
     ${CMAKE_DL_LIBS}
+    ${CMAKE_THREAD_LIBS_INIT}
+    ${EXPAT_LIBRARIES}
     )
 
 install(TARGETS fgrun RUNTIME DESTINATION bin)
